jkube-kit/resource/helm/src/main/java/org/eclipse/jkube/kit/resource/helm/ChartMuseumHelmRepositoryUploader.java

```diff
@@ -0,0 +1,46 @@
+/**
+ * Copyright (c) 2019 Red Hat, Inc.
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at:
+ *
+ *     https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *   Red Hat, Inc. - initial API and implementation
+ */
+package org.eclipse.jkube.kit.resource.helm;
+
+import org.eclipse.jkube.kit.common.KitLogger;
+
+import java.io.File;
+
+public class ChartMuseumHelmRepositoryUploader extends HelmRepositoryUploader {
+  private final KitLogger logger;
+
+  public ChartMuseumHelmRepositoryUploader(KitLogger logger) {
+    this.logger = logger;
+  }
+
+  @Override
+  public HelmRepository.HelmRepoType type() {
+    return HelmRepository.HelmRepoType.CHARTMUSEUM;
+  }
+
+  @Override
+  public String method() {
+    return "POST";
+  }
+
+  @Override
+  public KitLogger getLogger() {
+    return logger;
+  }
```

Comment:
Could simply be variables at HelmRespositoryUploader, no need to abstract-override them.